include/hermes/VM/Callable.h

```diff
@@ -705,8 +705,12 @@ class NativeFunction : public Callable {
 /// Object.
 class NativeConstructor final : public NativeFunction {
  public:
+  /// A CreatorFunction is responsible for creating the 'this' object that the
+  /// constructor function sees.
+  /// \p proto is the '.prototype' property of the constructor and should be set
+  /// as the __proto__ for the nascent object.
   using CreatorFunction =
-      CallResult<PseudoHandle<JSObject>>(Runtime *, Handle<JSObject>);
+      CallResult<PseudoHandle<JSObject>>(Runtime *, Handle<JSObject> proto);
```

Comment:
We should really fix this naming. In many places this is called `parentHandle` and the actual field is called `parent_`. Though in NativeConstuctor we also call it `prototype` and in other places `prototypeHandle`.